virtcontainers/network.go

```diff
@@ -579,6 +580,107 @@ func newNetwork(networkType NetworkModel) network {
 	}
 }
 
+const procMountInfoFile = "/proc/self/mountinfo"
+
+// getNetNsFromBindMount returns the network namespace for the bind-mounted path
+func getNetNsFromBindMount(nsPath string, procMountFile string) (string, error) {
+	netNsMountType := "nsfs"
+
+	// Resolve all symlinks in the path as the mountinfo file contains
+	// resolved paths.
+	nsPath, err := filepath.EvalSymlinks(nsPath)
```

Comment:
Does this build ?

Reply:
Yes It does. Why?

Reply:
I thought that using `:=` on the existing `nsPath` variable would not work, but I forgot that `err` was not declared yet!